test/controllers/ConfirmTransferControllerSpec.scala

```diff
@@ -431,18 +438,50 @@ class ConfirmTransferControllerSpec extends FrontEndTestHelper {
           .withCSRFToken
         ).futureValue
 
-      wiremockServer.getAllServeEvents.size() should equal(2)
+      wiremockServer.getAllServeEvents.size() should equal(3)
+    }
+  }
+
+  forAll(userChecks) { (user, url) =>
+    s"The $url upload page" should {
```

Comment:
Is the test description true now, given we are checking against the consignment type now, except on the series page?

Reply:
Ah no, I've changed it for this one and for all the other ones in the other files.